MonoidalCategories/gap/TerminalCategory.gi

```diff
@@ -326,8 +326,11 @@ InstallGlobalFunction( TerminalCategoryWithMultipleObjects,
     
     ## prevent skeletality and strictness
     properties := Set( List( CAP_INTERNAL_CATEGORICAL_PROPERTIES_LIST, a -> a[1] ) );
-    excluded_properties := Filtered( properties, p -> IsInt( PositionSublist( p, "Strict" ) ) );
-    Add( excluded_properties, "IsSkeletalCategory" );
+    excluded_strict_properties := Filtered( properties, p -> IsInt( PositionSublist( p, "Strict" ) ) );
```

Comment:
When porting this to #1009, please also use `ListImpliedFilters` when computing `excluded_strict_properties` to make this more future-proof.